CHANGELOG.md

```diff
@@ -5,6 +5,10 @@ This CHANGELOG follows the format listed [here](https://github.com/sensu-plugins
 
 ## [Unreleased]
 
+## [4.0.2] - 2018-03-30
```

Comment:
please remove version and date as they are done post acceptance by maintainers. This is for several reasons:
- PRs are not a FIFO queue, merge order is not guranteed
- there is no SLA on when reviews happen, we date the release not the submission as we might lump multiple submissions into the same release
- reduces the number of times we have to ask someone to rebase